doc/ideas/savefileformat.md

```diff
@@ -0,0 +1,446 @@
+# openage save format
+
+1.  [Questions](#1-questions)
+2.  [Overview](#2-overview)
+3.  [Header](#3-header)
+4.  [String Table](#4-string-table)
+5.  [Atrribute Table](#5-attribute-table)
+6.  [Game Information](#6-game-information)
+7.  [User Information](#7-user-information)
+8.  [Chunk Table](#8-chunk-table)
+9.  [Single Chunks](#9-single-chunks)
+10. [Triggers](#10-triggers)
+    1. [Event List](#101-event-list)
+    2. [Condition List](#102-condition-list)
+    3. [Action List](#103-action-list)
+11. [Crypto](#11-crypto)
+
+### 1 Questions
+
+Do we something like ingame time vs realtime if the game is played at a different speed?
```

Comment:
we just record keyframes. the timescale at which they are played afterwards doesn't matter.